### DISCUSSION TITLE
Creating an issue from a URL query

### DISCUSSION THREAD

**Discussion by juaram2** [community/community#33140] [Projects and Issues] [answered]
Hello, I'm trying to create an issue from a URL query

This is my query :
`&assignees=[USER]&template=[TEMPLATE]&labels=[LABEL]&projects=[ORG/NUMBER]`

`assignees`, `template`, `labels` all works but `projects` is not working.

I referred this : [Link](https://docs.github.com/en/enterprise-cloud@latest/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-a-url-query)
![image](https://user-images.githubusercontent.com/26706369/190335925-b9a2cbba-7f9e-4a1d-a85b-d962699c7626.png)

I tried to change "/" to "%2F". but failed.. How can I?

**Comment by dpasque** [comment]
Out of curiosity, in the project one of the new projects? (As in not a Classic project, but the projects that were formerly in beta?)

If so, I wonder if it's the same issue as this thread: https://github.com/orgs/community/discussions/19811

**Comment by rileybroughten** [comment, marked as answer]
Hello! 👋 This is not currently supported for Projects in comparison to Projects (classic), but there is another issue for the same request linked here https://github.com/community/community/discussions/19811 as @dpasque linked above.

I will close out this issue as a duplicate, and you can follow along on the above issue as updates will be provided in a single place! Thank you! ✨
